delivery_recipe/Delivery/steering_group_meetings.md

```diff
@@ -0,0 +1,27 @@
+# Steering Group Meetings
+
+## What is a steering group?
+
+A steering group is generally made up of senior managers within the client’s business. Steering groups are set up to provide direction to a project, programme of work or a constant organisational change.
```

Comment:
Maybe it's stakeholders rather than managers. We'd recommend a one team approach to steering group too, so the suppliers should be represented.